### DISCUSSION TITLE
ggplot2 boxplot问题

### DISCUSSION THREAD

**Discussion by zhaoliang0302** [ShixiangWang/self-study#47] [Q&A] [answered]
翔哥好，请教你关于boxplot中线条粗细的调整问题，我查了geom_boxplot这个函数没查到相应的设置。我画的图线条太粗了，由于x轴有很多变量，图再放大要超出A4范围，请问如何设置得细一点？谢谢
![image](https://user-images.githubusercontent.com/42333702/80691898-71491880-8b03-11ea-8406-d977c1e13e07.png)

**Comment by ShixiangWang** [comment, marked as answer]
调整 size 试试

```r
> p <- ggplot(mpg, aes(class, hwy))
> p + geom_boxplot()
> p + geom_boxplot(size = 2)
```

**Comment by zhaoliang0302** [comment]
完美解决，太感谢了
